docs/docusaurus/docs/core/customize_expectations/expectation_row_conditions.md

```diff
@@ -210,6 +210,7 @@ While conditions can be attached to most Expectations, the following Expectation
 
 * `expect_column_to_exist`
 * `expect_table_columns_to_match_ordered_list`
+* `expect_table_columns_to_match_set`
```

Comment:
**note for reviewer** I assume this limitation applies to this expectation as well since it's a Batch Expectation that is evaluated for an entire Batch

Reply:
Makes sense to me, and that looks like what I'm seeing in the code.